README.md

```diff
@@ -1,6 +1,8 @@
 # IndoNLU 
 ![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/indobenchmark/indonlu/blob/master/LICENSE) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
```

Comment:
Apparently, current `code_of_conduct.md` link is 404. please change code_of_conduct.md to https://github.com/indobenchmark/indonlu/blob/master/code_of_conduct.md. Nitip ubah ya. Thanks!

Reply:
Done!